tests/README.md

```diff
@@ -38,7 +53,21 @@ Parameters can use environmental variables with a formatted string where `"{<VAR
 ### Test Examples
 - **[blpop_test](./dragonfly/blpop_test.py)**: Simple test case interacting with Dragonfly
 - **[snapshot_test](./dragonfly/snapshot_test.py)**: Example test using `@dfly_args`, environment variables and pre-test setup
-- **[key_limt_test](./dragonfly/key_limit_test.py)**: Example test using `@dfly_multi_test_args`
+- **[key_limit_test](./dragonfly/key_limit_test.py)**: Example test using `@dfly_multi_test_args`
+- **[connection_test](./dragonfly/connection_test.py)**: Example for testing running with asynchronous multiple connections. 
+
+### Writing your own fixtures
+The fixture functions located in [conftest.py](./dragonfly/conftest.py).
+You can write your own fixture inside this file, as seem fit. Just make sure, before adding new fixture that there maybe one already writing.
+Try to make the fixture running at the smallest scape possible to ensure that the test can be independent of each other (this will ensure no side effect - match our policy of "share nothing").
```

Comment:
scape -> scope

Reply:
I didn't know that his word exist, this is why the spell check didn't find it :(